src/integTest/kotlin/io/snyk/plugin/services/SnykTaskQueueServiceHeavyTest.kt

```diff
@@ -0,0 +1,38 @@
+package io.snyk.plugin.services
+
+import com.intellij.testFramework.HeavyPlatformTestCase
+import com.intellij.testFramework.PlatformTestUtil
+import io.mockk.unmockkAll
+import io.snyk.plugin.getSnykTaskQueueService
+import io.snyk.plugin.removeDummyCliFile
+import io.snyk.plugin.resetSettings
+import org.junit.Test
+
+@Suppress("FunctionName")
+class SnykTaskQueueServiceHeavyTest : HeavyPlatformTestCase() {
+
+    override fun setUp() {
+        super.setUp()
+        unmockkAll()
+        resetSettings(project)
```

Comment:
Could you please mock Sentry here, so we don't get error reporting? Something like `mockkStatic(Sentry::class)`

Reply:
afaik no errors/exceptions from tests goes to Sentry, as the key (?) is deployed only into prod versions of the plugin